exploration/selection-declaration.md

```diff
@@ -1,6 +1,6 @@
 # Effect of Selectors on Subsequent Placeholders
 
-Status: **Proposed**
+Status: **Accepted**
```

Comment:
Let's not do this until we've agreed on a course to take. We can make this annotation when merging.